src/layouts/components/Editor/extensions/Details/IsomerDetailGroupView.tsx

```diff
@@ -13,94 +15,112 @@ export const IsomerDetailGroupView = ({
+                minW="1.75rem"
+                p="0.25rem"
+                aria-label="edit accordion block"
               >
-                <IconButton
-                  variant="outline"
-                  aria-label="Add accordion"
-                  icon={<Icon as={BiPlus} w="1.5rem" h="1.5rem" />}
-                  onClick={() => {
-                    editor.chain().appendDetail(startPos, endPos).run()
-                  }}
-                  position="absolute"
-                  bottom="0"
-                  left="50%"
-                  transform="translateX(-50%)"
-                  zIndex={2}
-                  background="white"
+                <Icon
+                  as={BiPencil}
+                  fontSize="1.25rem"
+                  color="base.content.medium"
                 />
-              </Box>
-              <Box
-                background="white"
-                zIndex={1}
-                right={0}
-                top={0}
-                mt="-1rem"
-                mr="0.5rem"
-                position="absolute"
-                minH="1.75rem"
+              </IconButton>
+            </Tooltip>
+            <Tooltip label="Delete accordion " hasArrow placement="top">
+              <IconButton
+                _hover={{ bg: "gray.100" }}
+                _active={{ bg: "gray.200" }}
+                onClick={() => {
+                  editor.chain().focus().unsetDetailsGroup().run()
+                }}
+                bgColor="transparent"
+                border="none"
                 h="1.75rem"
+                w="1.75rem"
+                minH="1.75rem"
                 minW="1.75rem"
-                color="base.content.medium"
-                borderColor="base.content.medium"
+                p="0.25rem"
+                aria-label="delete accordion block"
               >
-                <IconButton
-                  variant="outline"
-                  aria-label="Edit accordion"
-                  icon={<Icon as={BiPencil} w="1.25rem" h="1.25rem" />}
-                  onClick={() => {
-                    onDrawerOpen("accordion")()
-                  }}
-                  right={0}
-                  top={0}
-                  position="absolute"
-                  zIndex={2}
-                  background="white"
-                  minH="1.75rem"
-                  h="1.75rem"
-                  minW="1.75rem"
-                  color="base.content.medium"
-                  borderColor="base.content.medium"
+                <Icon
+                  as={BiTrash}
+                  fontSize="1.25rem"
+                  color="interaction.critical.default"
                 />
-              </Box>
-            </>
-          )}
-        </Box>
+              </IconButton>
+            </Tooltip>
+          </Box>
+        </HStack>
+      </Box>
+      <Box
+        position="absolute"
+        bottom="0"
+        left="50%"
+        transform="translateX(-50%)"
+        mb="-1.5rem"
```

Comment:
nit: idt `mb` works when position is `absolute`

Reply:
weirdly in this case it does leh 
![Screenshot 2023-12-28 at 10 30 09 AM](https://github.com/isomerpages/isomercms-frontend/assets/42832651/d9c3f77a-2d61-42b0-a92e-764f62c91ff9)

![Screenshot 2023-12-28 at 10 29 55 AM](https://github.com/isomerpages/isomercms-frontend/assets/42832651/6e114e7d-d688-4f85-b883-1169177236a3)

Reply:
oh cos we don't set `top`